php/libraries/NDB_BVL_Battery.class.inc

```diff
@@ -352,7 +352,32 @@ class NDB_BVL_Battery
         // return all the data selected
         return $rows;
     } // end getBatteryVerbose()
-
+    /**
+     * Checks to see if a table contains a specified column
+     *
+     * @param string $table_name The table to check for a column.
+     * @param string $column     The column name to check the table for.
+     *
+     * @return boolean true if the table has at least one NULL in the column
+     * @note   This should really be in the Database.class.inc file... -anyhowstep
```

Comment:
The comment is right, this belongs in the database class, not the battery class.